### ISSUE TITLE
Minimal setTimeout is 3000 ms

### ISSUE THREAD

**Issue by AlttiRi** [violentmonkey/violentmonkey#1163]
I have already noted this problem [here](https://github.com/Tampermonkey/tampermonkey/issues/1113#issuecomment-755044544), but it's better to create an issue. 

### What is the problem?
The minimal real setTimeout value is 3000 ms.

If a userscript uses `setTimeout` with 100 ms, the real timeout is 3 seconds. The script works _very_ slow especially if `setTimeout` is in a loop.

### How to reproduce it?
```js
// ==UserScript==
// @name          Slow setTimeout
// @namespace     GitHub
// @connect       https://testimages.juliaimages.org/
// @version       1.0.0
// @grant         GM_download
// @match         https://example.com/*
// ==/UserScript==

const urls = [ 'https://testimages.juliaimages.org/thumbnails/autumn_leaves.png',
               'https://testimages.juliaimages.org/thumbnails/blobs.png',
               'https://testimages.juliaimages.org/thumbnails/cameraman.png',
               'https://testimages.juliaimages.org/thumbnails/fabio_color_512.png',
               'https://testimages.juliaimages.org/thumbnails/earth_apollo17.png',
               'https://testimages.juliaimages.org/thumbnails/fabio_gray_256.png',
               'https://testimages.juliaimages.org/thumbnails/hela-cells.png',
               'https://testimages.juliaimages.org/thumbnails/lake_gray.png',
               'https://testimages.juliaimages.org/thumbnails/house.png',
               'https://testimages.juliaimages.org/thumbnails/pirate.png'];

(async function main() {
    let delay;
    for (const url of urls) {
        const name = url.match(/[^\/]+$/)[0];
        delay = sleep(100);
        await download(url, name);
        console.log("Downloaded");
        await delay;
    }
})();

function download(url, name) {
   return new Promise((resolve, reject) => {
       GM_download({
            url,
            name,
            saveAs: false,
            onerror: reject,
            onload: resolve
         });
   });
}

function sleep(ms) {
   return new Promise(resolve => setTimeout(resolve, ms));
}
```

### What is the expected result?
~0.1 seconds are between each `console.log`.
_(the result from TamperMonkey)_:
![_TM](https://user-images.githubusercontent.com/16310547/103725912-ae49a100-4fe8-11eb-875f-1db442414af9.png)

### What is the actual result?

![_VN](https://user-images.githubusercontent.com/16310547/103725797-6a569c00-4fe8-11eb-965e-4a2e2a0af371.png)
3 seconds are between each `console.log`.

### Environment
- Browser: Chromium 87
- Violentmonkey version: 2.12.9
- OS: Win10

**Comment by tophf** [comment]
Turns out Chrome ignores rapid `click` events on downloadable links (only in the active tab though) so I'll probably add throttling of 3 downloads per 500ms just like Tampermonkey did.

**Comment by AlttiRi** [comment]
> (only in the active tab though)

The background tabs' timers are throttled up to 1 additional second.
`sleep(100)`, `sleep(200)`, `sleep(500)` become `sleep(1000)`.
`sleep(1100)`, `sleep(1200)`, `sleep(1500)` become  `sleep(2000)`.

**Comment by tophf** [comment]
That's Chrome's doing. What I meant is that the original script downloaded all files if it runs in the inactive tab even without my new fix c123096.
